server/models/bhima.sql

```diff
@@ -481,9 +481,9 @@ INSERT INTO `analysis_tool_type` (`label`, `is_balance_sheet`, `rank`) VALUES
   ('FORM.LABELS.ANALYSIS_TOOLS.DEBTS', 1, 1);
 
 -- Cost Center basis info
-INSERT INTO `cost_center_basis` (`id`, `name`, `description`) VALUES
-  (1, 'ALLOCATION_BASIS_DIRECT_COST', 'ALLOCATION_BASIS_DIRECT_COST_DESCRIPTION'),
-  (2, 'ALLOCATION_BASIS_NUM_EMPLOYEES', 'ALLOCATION_BASIS_NUM_EMPLOYEES_DESCRIPTION'),
-  (3, 'ALLOCATION_BASIS_AREA_USED', 'ALLOCATION_BASIS_AREA_USED_DESCRIPTION');
+INSERT INTO `cost_center_basis` (`id`, `name`, `units`, `description`) VALUES
+  (1, 'ALLOCATION_BASIS_DIRECT_COST', '', 'ALLOCATION_BASIS_DIRECT_COST_DESCRIPTION'),
+  (2, 'ALLOCATION_BASIS_NUM_EMPLOYEES', '', 'ALLOCATION_BASIS_NUM_EMPLOYEES_DESCRIPTION'),
+  (3, 'ALLOCATION_BASIS_AREA_USED', 'm²', 'ALLOCATION_BASIS_AREA_USED_DESCRIPTION');
```

Comment:
We are supposed to have a UI for entering these information, for the name column it can be a problem to translate its value if on the handlebars template of the report we have something like :
```handlebars
{{translate 'FORM.' + name}}
```
And the user has to enter as he/she want the name of the cost center basis.

This table is perfect, but we have to write these name values in the root of translation files such as : 
```json
{
  "FORM":{...},
  "ALLOCATION_BASIS_DIRECT_COST":"....",
  ...
}
```

Reply:
Yes.  I've already thought of that.   I will work on that next.  My goal was to have some of the common ones available predefined with translations.  Then we can add more custom ones that will not have multiple translations.  I think what is in there can do both.